src/client/components/footer/footer.styles.css

```diff
@@ -0,0 +1,58 @@
+.footer {
+  background-color: #f8f8f8;
+  position: absolute;
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  height: 20%;
+}
+.footer ul {
+  display: flex;
+}
+.footer img {
+  margin-left: 10rem;
+}
+.footer ul li {
+  color: #666666;
+  list-style: none;
+  font-weight: 50;
+  font-size: 15px;
+  line-height: 12px;
+}
+.profile-icon {
+  position: absolute;
+  left: 60%;
+  right: 1.56%;
+  top: 11.13%;
+  bottom: 12.13%;
+  width: 20px;
+}
+.home-icon {
+  position: absolute;
+  left: 6.25%;
+  right: 5.21%;
+  width: 20px;
+  top: 11.13%;
+  bottom: 10.13%;
+}
+.profile-paragraph {
+  position: absolute;
+  left: 72%;
+  top: 40px;
+}
+.home-paragraph {
+  position: absolute;
+  left: 19%;
+  top: 40px;
+}
+@media screen and (max-width: 600px) {
```

Comment:
`/* Medium devices (landscape tablets, 768px and up) */
@media only screen and (min-width: 768px) {...}`
Nuha, code is looking perfect :)  but I am little bit confused about this Media Query. I mean for mobile and desktop version.

Reply:
it is responsive with mobile check the code in storybook and tell me

Reply:
good job nuha90

Reply:
I have checked the responsive style and changed for min-width and worked for me